pkg/ebpf/ingress_node_firewall_events.go

```diff
@@ -23,7 +29,8 @@ func (infc *IngNodeFwController) ingressNodeFwEvents() error {
 	// described in the eBPF C program.
 	rd, err := perf.NewReader(objs.IngressNodeFirewallEventsMap, os.Getpagesize())
 	if err != nil {
-		return fmt.Errorf("creating perf event reader: %s", err)
+		log.Printf("failed creating perf event reader: %s", err)
```

Comment:
Start message with uppercase

Reply:
nit - I'd go with `%q` as it will quote the error. otherwise, both %s and %v print the value.  +1 for starting upppercase
https://go.dev/play/p/-Dz3JOG28Va